src/Microsoft.AspNet.Mvc.Core/ViewComponents/ViewComponent.cs

```diff
@@ -173,6 +174,18 @@ public JsonViewComponentResult Json([NotNull] object value)
         }
 
         /// <summary>
+        /// Returns a result which will render JSON text.
+        /// </summary>
+        /// <param name="value">The value to output in JSON text.</param>
+        /// <param name="serializerSettings">The <see cref="JsonSerializerSettings"/> to be used by
+        /// the formatter.</param>
+        /// <returns>A <see cref="JsonViewComponentResult"/>.</returns>
+        public JsonViewComponentResult Json([NotNull] object value, [NotNull] JsonSerializerSettings serializerSettings)
```

Comment:
Why `[NotNull]` for `value`?  No such requirement in the `JsonViewComponentResult`